.teamcity/Gradle_Check/projects/StageProject.kt

```diff
@@ -62,14 +62,16 @@ class StageProject(model: CIBuildModel, stage: Stage, containsDeferredTests: Boo
             id = AbsoluteId(uuid)
             name = "Test coverage deferred from Quick Feedback and Ready for Merge"
             model.buildTypeBuckets
-                .filter { it.shouldBeSkippedInStage(stage) }
+                .filter { !it.shouldBeSkippedInStage(stage) && it.containsSlowTests() }
```

Comment:
According to https://github.com/gradle/gradle/blob/ddb80cb79cc882e42be704deb9b3d280db6e8aa5/.teamcity/Gradle_Check/projects/StageProject.kt#L66

Only `it.containsSlowTests()` is necessary here - we're trying to add slow test subproject into later stage.